draft-ietf-oauth-status-list.md

```diff
@@ -805,11 +839,11 @@ To avoid further correlatable information by the values of `uri` and `index`, Is
 - use decoy entries to obfuscate the real number of Referenced Tokens within a Status List
 - choose to deploy and utilize multiple Status Lists simultaneously
 
-## Third Party Hosting
+## Third Party Hosting {#third-party-hosting}
+
+If the roles of the Issuer and the Status Provider are performed by two different entities, this may give additional privacy assurances as the Issuer has no means to identity the Relying Party or its request.
```

Comment:
```suggestion
If the roles of the Issuer and the Status Provider are performed by two different entities, this may give additional privacy assurances as the Issuer has no means to identify the Relying Party or its request.
```